routes/flight.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+const FlightService = require('../services/flight-service.js')
+
+router.post('/seat-plan', async (req, res, next) => {
```

Comment:
here you should get the chosen seat id from the request, and modify the current flight accordingly — see if the user can actually reserve this seat.

so the method call should be `FlightService.chooseSeat(req.body)` instead of a generic `add` — we're not adding a flight seating plan, we're not the admin who are designing flights, we are customers who just want to book a predefined seat in the flight.